Packs/CTIX/Integrations/CTIX/CTIX.py

```diff
@@ -128,6 +141,21 @@ def get_domain_details(self, domain: list, enhanced: bool = False):
         client_url = self.base_url + url_suffix
         return self.http_request(full_url=client_url, **params)
 
+    def create_intel(self, data: dict):
+        """
+        Makes post call and creates Intel In CTIX Platform
+        :type data: ``dict``
+        :param data: Intel data
+
+        :return: dict containing post call response returned from the API
+        :rtype: ``Dict[str, Any]``
+        """
+        url_suffix = "create-intel/"
+        client_url = self.base_url + url_suffix
+        return self.http_request(
+            full_url=client_url, data=json.dumps(data), request_type="post"
```

Comment:
If you want, you can use the `json` argument. In this way you dont have to use `json.dumps()` and no need to set the headers content-type manually.